sceptre/template_handlers/file.py

```diff
@@ -50,16 +51,3 @@ def handle(self):
         except Exception as e:
             helper.print_template_traceback(path)
             raise e
-
-    def _resolve_template_path(self, template_path):
-        """
-        Return the project_path joined to template_path as
-        a string.
-
-        Note that os.path.join defers to an absolute path
-        if the input is absolute.
-        """
-        return path.join(
-            self.stack_group_config["project_path"], "templates",
-            normalise_path(template_path)
```

Comment:
wasn't this put in to make template handler work with windows paths?

Reply:
Perhaps @jfalkenstein could comment. At the time, I thought `normalise_path` was only required because I had moved some legacy code to the template handler class. Looking at it more closely, the new code may also have needed `normalise_path` to be applied to the `path` argument. What if I apply this patch here:

```diff
--- a/sceptre/template_handlers/file.py
+++ b/sceptre/template_handlers/file.py
@@ -4,7 +4,7 @@ import sceptre.template_handlers.helper as helper
 from pathlib import Path
 from sceptre.exceptions import UnsupportedTemplateFileTypeError
 from sceptre.template_handlers import TemplateHandler
-
+from sceptre.helpers import normalise_path
 
 class File(TemplateHandler):
     """
@@ -25,7 +25,7 @@ class File(TemplateHandler):
 
     def handle(self):
         project_path = self.stack_group_config.get("project_path")
-        input_path = Path(self.arguments["path"])
+        input_path = Path(normalise_path(self.arguments["path"]))
         if input_path.is_absolute():
             path = str(input_path)
         else:
```

Alternatively, I can just abort this part of the change and go with the version of the code that was already merged. However, I feel that I added unnecessary complexity.

Reply:
normalise_path takes a "sceptre_path" and turns it into an os-compatible path. This is important in the case of windows, where the user (per Sceptre's documentation) provides a template path like "directory/my-stack-config.yaml". Normalize_path will convert that to be windows-compatible. This is important for windows support.

Reply:
So yeah, I think we should probably still use it. Honestly... It's not clear to me why we're trying to put the code _back_ to it's previous state. The current state of this handler works well and accounts for all the edge cases we've discussed over the last week.

Reply:
@jfalkenstein Ok, how about we just add the tests on this branch.